src/System.ComponentModel/src/System.ComponentModel.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Also I see the same bug in System.Collections.Concurrent.csproj - please fix that file in another change too